core/src/com/unciv/logic/map/mapgenerator/MapRegions.kt

```diff
@@ -331,6 +344,12 @@ class MapRegions (val ruleset: Ruleset){
 
         // Next do negative bias ones (ie "Avoid []")
         for (civ in negativeBiasCivs) {
+            // If noStartBias is enabled consider these to be randomCivs
+            if (gameParameters.noStartBias) {
+                randomCivs.addAll(coastBiasCivs)
```

Comment:
wait wait don't merge! I just noticed I messed up here. Should have been addAll(negativeBiasCivs)